integrations/jira/.port/resources/blueprints.json

```diff
@@ -1,4 +1,83 @@
 [
+  {
+    "identifier": "jiraBoard",
+    "title": "Jira Board",
+    "description": "This blueprint represents a Jira board",
+    "icon": "Jira",
+    "schema": {
+      "properties": {
+        "url": {
+          "title": "Board URL",
+          "type": "string",
+          "format": "url",
+          "description": "URL to the board in Jira"
+        },
+        "type": {
+          "title": "Type",
+          "type": "string",
+          "description": "The type of the board"
+        }
+      },
+      "required": []
+    },
+    "mirrorProperties": {},
+    "calculationProperties": {},
+    "relations": {
+      "project": {
+        "target": "jiraProject",
+        "title": "Project",
+        "description": "The Jira project that contains this board",
+        "required": false,
+        "many": false
+      }
+    }
+  },
+
+  {
+    "identifier": "jiraSprint",
+    "title": "Jira Sprint",
+    "description": "This blueprint represents a Jira sprint",
+    "icon": "Jira",
+    "schema": {
+      "properties": {
+        "url": {
+          "title": "Sprint URL",
+          "type": "string",
+          "format": "url",
+          "description": "URL to the sprint in Jira"
+        },
+        "state": {
```

Comment:
Can we make this field as enum also if possible?
https://support.atlassian.com/jira-software-cloud/docs/sprint-states-in-advanced-roadmaps/